src/modules/feature-modules/assessment/pages/innovations/review-innovations.component.ts

```diff
@@ -115,6 +115,7 @@ export class ReviewInnovationsComponent extends CoreComponent implements OnInit
 
           return {
             ...item,
+            submittedAt: item.assessment?.reassessmentCount === 0 ? item.submittedAt : item.updatedAt,
```

Comment:
item.updated at is from the innovation and this value is changed every time the innovation is changed so it's not equivalent to the original submittedAt.

The closest thing to the submittedAt for reassessment is the item.assessment.createdAt (not this is most certainly not available yet in the innovations list)